<launch>
  <arg name="static_corner_point_tf" default="static_corner_point_tf"/>
  <arg name="linesegments_pub" default="line_segments"/>
  <arg name="corner_point_tolerance" default="5"/>
  <arg name="corner_point_x" default="9.0"/>
  <arg name="corner_point_y" default="-1"/>
  <arg name="scan" default="/edge/laser_front/scan"/>

  <node name="tuw_laser2corner_node" pkg="tuw_laser2corner" type="tuw_laser2corner_node" output="screen">
    <remap from="line_segments" to="$(arg linesegments_pub)"/>
    <param name="corner_point_tolerance" value="$(arg corner_point_tolerance)" type="double"/>
    <param name="corner_point_x" value="$(arg corner_point_x)" type="double"/>
    <param name="corner_point_y" value="$(arg corner_point_y)" type="double"/>
  </node>

  <include file="$(find tuw_linedetection)/launch/linesegment2d_detector_node.launch">
    <param name="laser_scan" value="$(arg scan)"/>
  </include>
</launch>
